tox.ini

```diff
@@ -9,7 +10,18 @@ deps =
 commands =
 	py.test {posargs}
 
+[testenv:negtz]
+passenv = LANG
+setenv =
+    TZ = America/Argentina/Buenos_Aires
+deps =
```

Comment:
I'm not really sure, but I suspect that these params could be inherited from `[testenv]`.